fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java

```diff
@@ -465,6 +470,20 @@ public CatalogIf replayCreateCatalog(CatalogLog log) throws DdlException {
         writeLock();
         try {
             CatalogIf catalog = CatalogFactory.constructorFromLog(log);
+            Map<String, String> props = log.getProps();
+            if (props.containsKey(METADATA_REFRESH_INTERVAL_SEC)) {
+                // need refresh
+                String catalogName = log.getCatalogName();
+                Integer metadataRefreshIntervalSec = null;
+                try {
+                    metadataRefreshIntervalSec = Integer.valueOf(props.get(METADATA_REFRESH_INTERVAL_SEC));
+                } catch (NumberFormatException e) {
+                    throw new DdlException("Invalid properties: " + METADATA_REFRESH_INTERVAL_SEC);
+                }
+                Integer[] sec = {metadataRefreshIntervalSec, metadataRefreshIntervalSec};
+                Env.getCurrentEnv().getRefreshManager().getRefreshMap().put(catalogName, sec);
```

Comment:
wrap the `getRefreshMap().put(catalogName, sec)` into `RefreshManger()`.
Better not expose raw map interface outside

Reply:
Ok ，I will modify the above code and encapsulation `getRefreshMap().put(catalogName, sec)` into `RefreshManger()`